test/compilable/imports/testlambda2.d

```diff
@@ -0,0 +1,3 @@
+module imports.testlambda2;
+
+int bar() { return 7; }
```

Comment:
I assume it's not smart enough to do:

```
enum seven = 7;
int bar() { return seven; }
```

yet?

Reply:
The body of the function which is called is not serialized. When `bar` is called in a lambda the mangled name of the function is put in the serialization + the serialization of the function arguments (the function body is not touched).